hazelcast/src/main/java/com/hazelcast/map/impl/MapMigrationAwareService.java

```diff
@@ -182,6 +185,20 @@ public void commitMigration(PartitionMigrationEvent event) {
                     event.getNewReplicaIndex());
         }
 
+        if (SOURCE == event.getMigrationEndpoint()) {
+            PartitionContainer partitionContainer
+                    = mapServiceContext.getPartitionContainer(event.getPartitionId());
+            for (RecordStore recordStore : partitionContainer.getAllRecordStores()) {
+                Set<MapOperation> offloadedOperations = recordStore.getOffloadedOperations();
```

Comment:
I am not familiar with the migration subsystem - does this also run on partition threads?

Reply:
yes, this part runs on partition threads.